### ISSUE TITLE
Add list of supported model

### ISSUE THREAD

**Issue by luohao123** [microsoft/BitNet#66]
The currently supported model appears rather scarce. Would it be considered to support a broader range of models?

**Comment by LeoX91** [comment]
They stated that with this release, their hope is that the community will train more 1bit models. So I do not think they'll release other models, even if I was hoping at least for florence and Phi.

**Comment by luohao123** [comment]
Yeah, flroence is good, but how to train 1bit model, is that need to further train? I actually hoping for florence3

**Comment by sd983527** [comment]
Please try with the latest model on HF. https://huggingface.co/microsoft/bitnet-b1.58-2B-4T-gguf
